extensions/wikia/AdEngine/js/spec/lookup/adapter/appnexus.spec.js

```diff
@@ -1,50 +1,56 @@
 describe('ext.wikia.adEngine.lookup.adapter.appnexus', function () {
 	'use strict';
 
-	function getAppNexus() {
-		return modules['ext.wikia.adEngine.lookup.adapter.appnexus'](
-			mocks.geo,
-			mocks.instantGlobals
-		);
-	}
-
 	var mocks = {
 		instantGlobals: {
 			wgAdDriverAppNexusBidderCountries: ['PL']
 		},
 		geo: {
 			isProperGeo: jasmine.createSpy('isProperGeo')
+		},
+		appNexusPlacements: {
+			getPlacement: function () {
+				return '123';
```

Comment:
it's a mock, what value would you like to have here? it doesn't have to be "production-like" data, it should be similar, doesn't have to be the same.